mscore/file.cpp

```diff
@@ -1850,6 +1852,10 @@ bool MuseScore::saveAs(Score* cs, bool saveCopy, const QString& path, const QStr
             // save as MusicXML *.xml file
             rv = saveXml(cs, fn);
             }
+      else if (ext == "musicxml") {
```

Comment:
can't we make this `if (ext == "xml" || ext == "musicxml")` to avoid code duplication?